.github/workflows/coordinator-test.yml

```diff
@@ -1,4 +1,4 @@
-name: CI IT Experiment
+name: Coordinator Tests
```

Comment:
Can define a better name but these are tests for main-level projects; for v1 it's all components (since APIs part of Coordinator), for v2 just Coordinator tests (cql, grpc)

Reply:
I like this name